### PR TITLE
Expose CDI Upload Proxy service

### PR DESCRIPTION
**Pull request by igoihman**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
Added Openshift Route to expose CDI Upload Proxy service
```

### CODE REVIEW

**Comment by igoihman** [comment]
@gbenhaim @rthallisey could you review?

File: roles/cdi/tasks/provision.yml [hunk 1]
```diff
@@ -40,3 +40,11 @@
...
 - name: Wait for CDI operator to report that everything is running
   command: "{{ cluster_command }} -n {{ cdi_namespace }} wait cdis.cdi.kubevirt.io/cdi --for=condition=running --timeout=300s"
+
+- name: Render CDI Upload Proxy Route
+  template:
+    src: "cdi-uploadproxy-route.yaml.j2"
+    dest: "/tmp/cdi-uploadproxy-route.yaml"
+
+- name: Create CDI Upload Proxy Route
+  command: "{{ cluster_command }} apply -f /tmp/cdi-uploadproxy-route.yaml --validate=false"
```
Comment (rthallisey) [inline roles/cdi/tasks/provision.yml]: Can you move this before ```- name: Render CDI operator resources``` so this gets time to launch during the wait.

**Comment by rthallisey** [comment]
@igoihman One minor change.  

CI is breaking on https://github.com/kubevirt/kubevirt-ansible/issues/575, so it's passing CDI tests and is good to merge after the change.

**Comment by rthallisey** [comment]
LG2M.  CI is failing on an unrelated test https://github.com/kubevirt/kubevirt-ansible/issues/575

**Pull request merged**